keyboards/dz60/keymaps/nazu/config.h

```diff
@@ -0,0 +1 @@
+#define DYNAMIC_KEYMAP_LAYER_COUNT 5
```

Comment:
This needs an include guard: 

```suggestion
#pragma once

#define DYNAMIC_KEYMAP_LAYER_COUNT 5
```